securitycenter/snippets_v2/snippets_notification_v2_test.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+#
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for snippets."""
+
+import os
+import uuid
+
+import backoff
+from google.api_core.exceptions import InternalServerError, NotFound, ServiceUnavailable
+from google.cloud import securitycenter_v2 as securitycenter_v2
+import pytest
+
+import snippets_notification_configs_v2
+
+ORG_ID = os.environ["GCLOUD_ORGANIZATION"]
+PROJECT_ID = os.environ["GCLOUD_PROJECT"]
+PUBSUB_TOPIC = os.environ["GCLOUD_PUBSUB_TOPIC"]
+PUBSUB_SUBSCRIPTION = os.environ["GCLOUD_PUBSUB_SUBSCRIPTION"]
+LOCATION_ID = os.environ["GCLOUD_LOCATION"]
+
+CREATE_CONFIG_ID = "new-notification-pytest" + str(uuid.uuid1())
+DELETE_CONFIG_ID = "new-notification-pytest" + str(uuid.uuid1())
+GET_CONFIG_ID = "new-notification-pytest" + str(uuid.uuid1())
+UPDATE_CONFIG_ID = "new-notification-pytest" + str(uuid.uuid1())
+
+
+def cleanup_notification_config(notification_config_id):
+    client = securitycenter_v2.SecurityCenterClient()
+
+    notification_config_name = (
+        "organizations/{org_id}/locations/{location_id}/notificationConfigs/{config_id}".format(
+            org_id=ORG_ID, location_id=LOCATION_ID, config_id=notification_config_id
+        )
+    )
+    client.delete_notification_config(request={"name": notification_config_name})
+
+
+@pytest.fixture
+def new_notification_config_for_update():
```

Comment:
Would it be possible to use a single notification config to test all the CRUD samples, instead of creating one for each test?

Also, in the assertion statements, kindly assert if the specific config name is returned.

Reply:
We cannot do single notification config for all the test cases. Updated the assertion statement.

Reply:
@thokalavinod I don't see the value in duplicating code to create multiple notification configs. I strongly suggest you remove it. Refer to the Java test file here: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/security-command-center/snippets/src/test/java/vtwo/NotificationIT.java

@msampathkumar Kindly confirm if this is possible in python.

Reply:
addressed

Reply:
@Sita04 Good point!

We are looking for code coverage & at least 1 test for each region tag code sample.

@thokalavinod - I see & appreciate you work but in future, it could be difficult to manage complex test cases. So, as Sita suggested, would it be possible to simplify this test file?